README.md

```diff
@@ -73,6 +73,7 @@ plot(sf::st_geometry(streets),
 Determine continuous lines in the network as:
 
 ``` r
+library(rcoins)
 continuous_streets <- stroke(streets)
 ```
 
```

Comment:
Since `CRiSpData` is cropped, the figure in the readme `man/figures/continuous-streets.png` should be updated too, but the difference is minor.